application/simple/templates/index.html

```diff
@@ -308,6 +308,39 @@
             output of <code>conda list</code> inside the activated environment.
           </p>
 
+          <h2>Upload your environment to anaconda.org</h2>
+
+          <p>
+            Now that you have your <code>environment.yml</code> ready, you can upload
+            it to <a href="http://anaconda.org">anaconda.org</a>. You will first need
+            an account there. Go ahead, it's free!
+          </p>
+
+          <p>
+            With your <a href="http://anaconda.org">anaconda.org</a>'s credentials you
+            are ready to upload your environment with:
+          </p>
+
+          {% call code.block("bash") %}
+$ conda env upload name-of-the-environment
```

Comment:
According to the documentation style guide, we should avoid putting the `$` in front of commands.  It sometimes confuses users, and also makes cut & paste harder.